react-api-client/src/runs/useAddCapturePreviewImageToRun.ts

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from 'react-query'
+
+import { addCapturePreviewImageToRun } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type { UseQueryOptions, UseQueryResult } from 'react-query'
+import type {
+  CameraImageSettings,
+  DownloadedImageFileResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export function useAddCapturePreviewImageToRun(
+  cameraImageSettings: CameraImageSettings,
+  runId: string,
+  options?: UseQueryOptions<
+    DownloadedImageFileResponse,
+    AxiosError<ErrorResponse>
+  >
+): UseQueryResult<DownloadedImageFileResponse, AxiosError<ErrorResponse>> {
+  const host = useHost()
+
+  return useQuery<DownloadedImageFileResponse, AxiosError<ErrorResponse>>(
```

Comment:
```suggestion
  return useMutation...(
```
Whoops, I missed this in the last PR too, but because the underlying network request is a `POST` request, we want a mutation here. `useQuery` is for `GET`s. So yeah, we'll need to update the global image preview hook, too.

Like our other mutation hooks, we should return our own `capturePreviewImageForRun`, too. Same goes for the global hook.